src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/hooks/useCreateDrawerForm.tsx

```diff
@@ -257,8 +275,8 @@ const useCreateDrawerForm = (
   const onChangeStartVM = (checked: boolean) => {
     setVM(
       produce(vm, (draftVM) => {
-        delete draftVM.spec.runStrategy;
-        draftVM.spec.running = checked;
+        delete draftVM.spec.running;
+        draftVM.spec.runStrategy = checked ? 'Always' : 'Halted';
```

Comment:
please export to a const file the Always, Halted and ReturnOnFailure then consume it where needed

Reply:
done.